types/d3/v3/index.d.ts

```diff
@@ -2576,7 +2577,7 @@ declare namespace d3 {
             tickPadding(): number;
             tickPadding(padding: number): Axis;
 
-            tickFormat(): (t: any) => string;
+            tickFormat(): (t: any , index ?: number) => string;
```

Comment:
1. The signature for this function should be `(t: any, index: number) => string` - we can't guarantee that the function doesn't use the second parameter. (If you know it doesn't, you can use a type assertion to ignore it.)
2. Make sure you're reflecting this extra parameter in the setter below this one. The tests are failing because the two-argument function you passed in don't match the declaration.

Reply:
Thanks for the feedback. Changes made.